webapp/src/app.tsx

```diff
@@ -128,7 +128,7 @@ const App = React.memo((): JSX.Element => {
     }
 
     const continueToWelcomeScreen = () => {
-        return Utils.isFocalboardPlugin() && loggedIn === true && !UserSettings.welcomePageViewed
+        return Utils.isFocalboardPlugin() && loggedIn === true && !me?.props[UserSettingKey.WelcomePageViewed]
```

Comment:
Also unrelated?  Is this PR based on another PR that is unmerged?

Reply:
Nope, totally related. We're moving `WelcomePageViewed` from being managed in browser localstorage to server side user props.